jdbc/pom.xml

```diff
@@ -74,8 +74,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
-      <version>2.7.2</version>
-      <scope>provided</scope>
+      <version>${hadoop.common.version}</version>
     </dependency>
```

Comment:
Are you asking about renaming the key name from `hadoop.common.version` to `hadoop.version` or instead of this key use directly `hadoop.version` ?